src/Storage/TSDatabaseView.h

```diff
@@ -14,17 +14,14 @@ extern NSString *TSSecondaryDevicesGroup;
 
 extern NSString *TSThreadDatabaseViewExtensionName;
 extern NSString *TSMessageDatabaseViewExtensionName;
-extern NSString *TSThreadIncomingMessageDatabaseViewExtensionName;
```

Comment:
I just iterate in TSMessageDatabaseViewExtensionName now instead of using this view.